file_service/management/commands/migrate_files.py

```diff
@@ -0,0 +1,55 @@
+import logging
+import ssl
+if hasattr(ssl, '_create_unverified_context'):
+    ssl._create_default_https_context = ssl._create_unverified_context
+
+from optparse import make_option
+
+from django.core.management.base import BaseCommand, CommandError
+from django.core.management import call_command
+
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = 'Imports iSites file repository export to Canvas'
```

Comment:
Should this read 'Migrate iSites file ....'  to distinguish it from  the help in import_files?